src/zimdump.cpp

```diff
@@ -296,27 +319,51 @@ void ZimDumper::listArticleT(const zim::Article& article, bool extra)
   }
   std::cout << std::endl;
 }
+void write_to_error_directoy(const std::string& base, const std::string relpath, const char *content, size_t size)
+{
+    createdir(ERRORSDIR, base);
+    std::string url = relpath;
+
+    std::string::size_type p;
+    while ((p = url.find('/')) != std::string::npos)
+        url.replace(p, 1, "%2f");
+
+    std::ofstream stream(base + ERRORSDIR + url);
```

Comment:
The `base` string contains the namespace (`<currentdir>/A/`).
It means that the error directory will be something like `<currentdir>/A/_errordir/`. It should be `<currentdir>/_errordir/A`

Reply:
@mgautierfr `base` string does not contain the namespace. It comes directly from the program argument.

Reply:
Isn't set with the namespace here : https://github.com/openzim/zim-tools/blob/create_dir_structure_when_dumping/src/zimdump.cpp#L381-L382 ?

Reply:
My bad, you are using `directory + SEPARATOR` as argument. We are good here.